docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM grundrausch3n/ergw-docker-base
```

Comment:
I would prefer if you started with a plain release image from Ubuntu/Centos/CoreOS/Whatever release and not with a prebuild image. This way the steps to create the final image are much more transparent and if anyone wants/needs to adjust them, its much simpler for them.